console/billing/quickstart.mdx

```diff
@@ -0,0 +1,76 @@
+
+Before you can order Scaleway resources, you must add your payment method to your account.
+
+<Macro id="iam-requirements" />
+
+1. Click **Billing** in the Organization drop-down menu to go to the [Billing page](https://console.scaleway.com/organization/billing).
+2. Click **Payment and billing**.
+3. Scroll down to **Payment Methods**.
+
+You can choose between two payment methods: **credit card** or **SEPA mandate**.
+
+### How to add a credit card
+
+1. Enter the details of your credit or debit card.
+2. Click **Add credit card** to confirm. A [3D Secure](https://en.wikipedia.org/wiki/3-D_Secure) validation is initiated. Depending on your bank, you may be asked to enter a code received by SMS or via an application on your smartphone.
+3. Enter the code and validate the form.
+    <Message type="note">
+     When you register a new credit card, an authorization charge of **1€** is debited from your account. This **Authorization Charge** is a dummy transaction made to check the validity of the card and will carry a 4-digit validation code, displayed on your bank account statement. The authorization charge will be reimbursed within 48 to 72 hours.
+    </Message>
+4. Click **Verify**. A pop-up appears.
+5. Enter the 4-digit validation code displayed on your bank account statement.
+    <Message type="note">
+      Bank account statements have different displays depending on the bank, but the validation code can always be found in the description of your transactions. The image below provides an example of how the validation code might appear in your bank statement:
+
+      <Lightbox src="scaleway-bank-statement.webp" alt="Screenshot of a bank account statement. Under the transactions section, an entry called SCW*C-6569E-COMMERCE displays. The validation code is represented by the number sequence" />
+    </Message>
+
+
+<Message type="important">
+  You have up to 10 days to enter your validation code in the console.
+</Message>
+
+### How to add a SEPA mandate
+
+<Message type="note">
+  This method requires a successful [KYC verification](/console/account/how-to/verify-identity).
+</Message> 
+
+1. Click **SEPA Direct Debit** under **Payment Methods**.
+2. Click **+Add a SEPA mandate**. A pop-up appears.
+3. Enter the required information.
+4. Click **Add a SEPA mandate** to confirm.
+
+### How to change your billing information
+
+1. Click **Billing** in the Organization drop-down menu to go to the [Billing page](https://console.scaleway.com/organization/billing).
+2. Click <Icon name="edit" /> in the **Billing information** area. A pop-up displays.
+3. Enter your new postal address and click **Confirm changes** to validate.
+
+    Your new billing address is now used for your invoices.
+
+<Message type="note">
+  It is not possible to edit your country and region information directly from this form. Contact the [support team](https://console.scaleway.com/support/tickets/) if you need to modify this information.
+</Message>
+
```

Comment:
In our files, we usually don't add spaces after the end of the content. But to double-check with the TWs. :)